### DISCUSSION TITLE
Flap settings

### DISCUSSION THREAD

**Discussion by martin3432** [junzis/openap#36] [Ideas]
Hi Junzi,
I and my team would like to use OpenAP for our trajectory optimization tool TOMATO.

For the calculation of drag for a nonclean aircraft configuration, a flap angle in degree is required. However, in the aircraft.yml we miss the aircraft-specific flap positions. Is there a reason for this?

I can add the flap positions for some aircraft by scanning the FCOM Manuals, which can be partly found on the Internet. I suggest this scheme in aircraft.yml:

`flaps:`
&ensp;`type:` ...
&ensp;`positions:`
&ensp;&ensp;&ensp;`1+F:` 10
&ensp;&ensp;&ensp;`2:` 15

Is this an option for you?

**Comment by junzis** [comment]
Hi @martin3432, sorry for the very late reply. I just saw this message.

Yes, that would be great! Please feel free to change that and make a pull request. I will merge that in the main branch.

Thanks!
